website/releases/download.md

```diff
@@ -7,14 +7,17 @@ last_modified_at: 2022-12-27T15:59:57-04:00
 ---
 
 ### Release 0.12.2
+* [Long Term Support](/releases/release-0.12.2#long-term-support): this is the latest stable release
 * Source Release : [Apache Hudi 0.12.2 Source Release](https://www.apache.org/dyn/closer.lua/hudi/0.12.2/hudi-0.12.2.src.tgz) ([asc](https://downloads.apache.org/hudi/0.12.2/hudi-0.12.2.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.12.2/hudi-0.12.2.src.tgz.sha512))
```

Comment:
For long term, does it mean we would have 0.12.3, or even 0.12.4 minor releases ?

Reply:
Yes.